src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -1274,16 +1274,16 @@
     <Kind Name="AnonymousObjectMemberDeclarator"/>
     <Field Name="NameEquals" Type="NameEqualsSyntax" Optional="true">
       <PropertyComment>
-        <summary>NameEqualsSyntax representing the optional name of the property being initialized.</summary>
+        <summary>NameEqualsSyntax representing the optional name of the member being initialized.</summary>
```

Comment:
I'm not sure but either is fine. At the moment we only support initialization of fields and properties but I know there is a feature request to support initializing events as well so member might be better long term. On the other hand I prefer being more specific if possible. Again, either way is fine by me.

Regards,

-ADG

Sent from my Windows Phone
________________________________
From: Paul Harrington<mailto:notifications@github.com>
Sent: ‎5/‎14/‎2015 18:58
To: dotnet/roslyn<mailto:roslyn@noreply.github.com>
Cc: Anthony D. Green<mailto:adgreen@microsoft.com>
Subject: Re: [roslyn] Minor cleanup of C#'s Syntax.xml (#2750)


In src/Compilers/CSharp/Portable/Syntax/Syntax.xml<https://github.com/dotnet/roslyn/pull/2750#discussion_r30375392>:

> @@ -1274,16 +1274,16 @@
>      <Kind Name="AnonymousObjectMemberDeclarator"/>
>      <Field Name="NameEquals" Type="NameEqualsSyntax" Optional="true">
>        <PropertyComment>
> -        <summary>NameEqualsSyntax representing the optional name of the property being initialized.</summary>
> +        <summary>NameEqualsSyntax representing the optional name of the member being initialized.</summary>


Do we use the term 'member' elsewhere? Or should we say "field or property" (assuming that's what it means here)?
@AnthonyDGreen<https://github.com/AnthonyDGreen> ?

—
Reply to this email directly or view it on GitHub<https://github.com/dotnet/roslyn/pull/2750/files#r30375392>.